joomla-framework/github-api
# Provide access_token in header instead of URI

Pull Request for Issue #48

### Summary of Changes

Adds the `authHeader()` method to \Joomla\Github\AbstractGithubObject.
Changes unit tests to ensure the required treatment of header information.

### Testing Instructions

Verify that the changes in the unit tests will test the right things.
Run PHPUnit.

### Documentation Changes Required

API documentation will be updated automatically.
